### DISCUSSION TITLE
Free Podcast Transcription Desktop from Spreaker

### DISCUSSION THREAD

**Discussion by lele85** [tauri-apps/tauri#6630] [Show and tell]
https://user-images.githubusercontent.com/49144/229761445-07409237-3a04-46ec-a334-4afac7c3d11d.mov

Some weeks ago at Spreaker we launched https://freepodcasttranscription.com/ using WASM and whisper.cpp. The tool is quite simple in term of functionality and WASM is pretty decent with small models but i was still not super satisfied with the performance.

What a better chance to test `Tauri`? In just a week we did a desktop porting and now is out with a build for `Apple Silicon` and `Apple Intel`:

### Apple Silicon:
https://freepodcasttranscription.com/desktop/darwin/aarch64/Free%20Podcast%20Transcription.dmg

### Apple Intel
https://freepodcasttranscription.com/desktop/darwin/x86_64/Free%20Podcast%20Transcription.dmg

Let me give my 2 cents coming from an `electron` background:

- Tauri feels at least 10 times better than electron
- The IPC model makes a lot of sense, even if there are things that are a little more convoluted
- Rust is cool, never used it before, but with a little help from copilot i was able to do some neat things (like calling directly into C and using sidecar binaries)

All of this to say that i really enjoyed using Tauri, and i'm grateful for what y'all are building :)

**Comment by Manubi** [comment]
@lele85 is there a chance I can see the code as I would like to play with it as well? 
Or if no code... can you give me some help on how you did it. thanks
